docs/content/2.deploy/providers/cleavr.md

```diff
@@ -0,0 +1,31 @@
+---
+title: Cleavr
+description: 'Discover Cleavr preset for Nitro!'
+---
+
+## Cleavr
+
+**Preset:** `cleavr` ([switch to this preset](/deploy/#changing-the-deployment-preset))
+
+Nitro provides a built-in preset to generate output format compatible with [Cleavr.io](https://www.cleavr.io/).
```

Comment:
www. subdomains seems broken. It redirects me to https://country-lookup.cleaver.rocks/